src/txpasslib/_threads.py

```diff
@@ -0,0 +1,24 @@
+from twisted.internet.defer import Deferred
+from twisted.python.failure import Failure
+
+
+def defer_to_worker(deliver, worker, work, *args, **kwargs):
+    """
+    Run a task in a worker, delivering the result as a ``Deferred`` in the
```

Comment:
I'm not going to change all the docstrings in this branch because a) I'm lazy, and b) we don't have any API docs thing set up yet, so I can't actually test my syntax anyway. Once we set something up I'll change all the docstrings in that branch.